### PR TITLE
Parse out maxmem-profile info from relval logs and write to a file

### PR DESCRIPTION
**Pull request by gartung**

### CODE REVIEW

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
please test

**Comment by gartung** [comment]
I picked this scripts because it is the one run in this Jenkins job
https://cmssdt.cern.ch/jenkins/job/ib-run-pr-relvals/

**Comment by gartung** [comment]
The [ib-run-relvals Jenkins job](https://cmssdt.cern.ch/jenkins/job/ib-run-relvals/) runs `run-ib-relval.py`

**Comment by smuzaffar** [comment]
> The [ib-run-relvals Jenkins job](https://cmssdt.cern.ch/jenkins/job/ib-run-relvals/) runs `run-ib-relval.py`

this is for threaded relvals of IBs. 
 https://github.com/cms-sw/cms-bot/blob/master/run-ib-pr-matrix.sh isused by both PR relvals and baseline relvals (both non-threaded)

**Comment by gartung** [comment]
please test

File: run-ib-pr-matrix.sh [hunk 1]
```diff
@@ -78,6 +78,17 @@ pushd "$WORKSPACE/matrix-results"
     fi
   done
   set -x
+
+  LOG=runall-report-step123-.${BUILD_ID}.log
```
Comment (smuzaffar) [inline run-ib-pr-matrix.sh:82]: @gartung , I would suggest to run tis only if `$CMD_OPTS` container `maxmem_profile` option e.g
```
if [[ $CMD_OPTS =~ maxmem_profile ]] ; then
  #do the processing....
fi
```

Comment (gartung) [reply run-ib-pr-matrix.sh]: OK

**Comment by gartung** [comment]
please test

**Pull request merged**